lib/ejs.js

```diff
@@ -52,6 +52,7 @@ var scopeOptionWarned = false;
 var _VERSION_STRING = require('../package.json').version;
 var _DEFAULT_DELIMITER = '%';
 var _DEFAULT_LOCALS_NAME = 'locals';
+var _NAME = 'ejs';
```

Comment:
I think the underscorized constant makes sense, since the version string is in the same place. I might have declared it right next to the `_VERSION_STRING`, but this works.